frontend/src/views/Dashboard.vue

```diff
@@ -117,13 +134,14 @@ export default {
         a-empty(v-if="user && !user.events.length")
           span(slot="description") You haven't created any events. <br> Click the button below to create your first event 🎉
           a-button(type="primary" @click="focusOnCreateEventForm" icon="plus") Create Event
-        a-card(v-for="event in sortedEvents" :key="event.id")
+        a-card(v-for="event in paginatedEvents[pageIndex]" :key="event.id")
           h3
             router-link(:to="`/events/${event._id}`") {{ event.title }}
           p {{ event.description }}
           p
             a-icon(type="message")
             span &nbsp{{ event.questions.length }}
+        a-pagination(v-if="user && user.events.length" :total="user.events.length" :default-page-size="paginatedItemCount" @change="(page) => pageIndex = page - 1")
```

Comment:
instead of the change event we should keep a property called `currentPage` and bind it to the `current` attribute. This means wherever we use pageIndex right now, we should use `currentPage - 1`